
Showing logs for build [32mseldonio-seldon-core-pr-1406-li-hvqdp-2[0m stage [32mpr-build-comment[0m and container [32mstep-credential-initializer-cz68r[0m
{"level":"warn","ts":1580843158.597292,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1580843158.598446,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}

Showing logs for build [32mseldonio-seldon-core-pr-1406-li-hvqdp-2[0m stage [32mpr-build-comment[0m and container [32mstep-working-dir-initializer-9zc5f[0m
{"level":"warn","ts":1580843159.6255815,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1580843159.6322162,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1406-li-hvqdp-2[0m stage [32mpr-build-comment[0m and container [32mstep-place-tools[0m

Showing logs for build [32mseldonio-seldon-core-pr-1406-li-hvqdp-2[0m stage [32mpr-build-comment[0m and container [32mstep-create-dir-workspace-qr2c4[0m
{"level":"warn","ts":1580843168.7085843,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1580843168.7141712,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1406-li-hvqdp-2[0m stage [32mpr-build-comment[0m and container [32mstep-git-source-seldonio-seldon-core-pr-1406-li-xtd5b[0m
{"level":"warn","ts":1580843169.307551,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1580843173.126262,"logger":"fallback-logger","caller":"git/git.go:103","msg":"Successfully cloned https://github.com/SeldonIO/seldon-core.git @ master in path /workspace/source"}

Showing logs for build [32mseldonio-seldon-core-pr-1406-li-hvqdp-2[0m stage [32mpr-build-comment[0m and container [32mstep-git-merge[0m
Using SHAs from PULL_REFS=master:3763cc66ca1a8619a00dcdc3a46a2eef780a39bf,1406:5cb7b4d83c8cf392c7b672633b5f7b9987d70e5e
DEBUG: ran git fetch --unshallow origin 5cb7b4d83c8cf392c7b672633b5f7b9987d70e5e: 3763cc66ca1a8619a00dcdc3a46a2eef780a39bf: in 
DEBUG: ran git checkout master in 
DEBUG: ran git reset --hard 3763cc66ca1a8619a00dcdc3a46a2eef780a39bf in 
DEBUG: ran clean --force -d . in 
DEBUG: ran git merge 5cb7b4d83c8cf392c7b672633b5f7b9987d70e5e in 
Merged SHA 7196334bcf98cb1e316239ff580db9b38f45a50d with commit message 'Merge commit '5cb7b4d83c8cf392c7b672633b5f7b9987d70e5e'' into base branch master
Merged SHA 5cb7b4d83c8cf392c7b672633b5f7b9987d70e5e with commit message 'Add timeout and message size annotations and tests' into base branch master
Merged SHA ef7052c9d5af7e5a35b60bff51b8399392fe52a5 with commit message 'Added processing of annotations and grpc message size in particular' into base branch master

Showing logs for build [32mseldonio-seldon-core-pr-1406-li-hvqdp-2[0m stage [32mpr-build-comment[0m and container [32mstep-step2[0m
+++ dirname ./add-pr-build-comment
++ cd .
++ pwd
+ STARTUP_DIR=/workspace/source/ci
+ REPO_OWNER=SeldonIO
+ REPO_NAME=seldon-core
+ PULL_NUMBER=1406
+ BUILD_NUMBER=2
+ PIPELINE_CONTEXT=lint
++ cat
+++ date
+ COMMENT_MSG='Tue Feb  4 19:06:20 UTC 2020
The logs for [lint] [2] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1406/2.log

impatient try  
** get build logs SeldonIO/seldon-core/PR-1406 --build=2'
+ ** step pr comment --owner=SeldonIO --repository=seldon-core --pull-request=1406 '--comment=Tue Feb  4 19:06:20 UTC 2020
The logs for [lint] [2] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1406/2.log

impatient try  
** get build logs SeldonIO/seldon-core/PR-1406 --build=2'

Showing logs for build [32mseldonio-seldon-core-pr-1406-li-hvqdp-2[0m stage [32mpr-build-comment[0m and container [32mstep-source-mkdir-seldonio-seldon-core-pr-1406-li-mws68[0m
{"level":"warn","ts":1580843182.94123,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1580843182.9516697,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /pvc/pr-build-comment/workspace\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1406-li-hvqdp-2[0m stage [32mpr-build-comment[0m and container [32mstep-source-copy-seldonio-seldon-core-pr-1406-li-zn5hd[0m
{"level":"warn","ts":1580843183.522262,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1580843184.0397356,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c cp -r source/. /pvc/pr-build-comment/workspace\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1406-li-hvqdp-2[0m stage [32mpython-lint-task[0m and container [32mstep-credential-initializer-hf9cf[0m
{"level":"warn","ts":1580843222.6671588,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1580843222.667675,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}

Showing logs for build [32mseldonio-seldon-core-pr-1406-li-hvqdp-2[0m stage [32mpython-lint-task[0m and container [32mstep-working-dir-initializer-hd8fl[0m
{"level":"warn","ts":1580843223.6130028,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1580843223.6161764,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1406-li-hvqdp-2[0m stage [32mpython-lint-task[0m and container [32mstep-place-tools[0m

Showing logs for build [32mseldonio-seldon-core-pr-1406-li-hvqdp-2[0m stage [32mpython-lint-task[0m and container [32mstep-create-dir-workspace-78nqm[0m
{"level":"warn","ts":1580843228.7589889,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1580843228.7618322,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1406-li-hvqdp-2[0m stage [32mpython-lint-task[0m and container [32mstep-source-copy-workspace-xdx4n[0m
{"level":"warn","ts":1580843229.1050427,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1580843232.2070222,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c cp -r /pvc/pr-build-comment/workspace/. /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1406-li-hvqdp-2[0m stage [32mpython-lint-task[0m and container [32mstep-python-lint[0m
make: Entering directory '/workspace/source/python'
pip install black==19.3b0
Collecting black==19.3b0
  Downloading https://files.pythonhosted.org/packages/30/62/cf549544a5fe990bbaeca21e9c419501b2de7a701ab0afb377bc81676600/black-19.3b0-py36-none-any.whl (89kB)
Collecting attrs>=18.1.0
  Downloading https://files.pythonhosted.org/packages/a2/db/4313ab3be961f7a763066401fb77f7748373b6094076ae2bda2806988af6/attrs-19.3.0-py2.py3-none-any.whl
Collecting click>=6.5
  Downloading https://files.pythonhosted.org/packages/fa/37/45185cb5abbc30d7257104c434fe0b07e5a195a6847506c074527aa599ec/Click-7.0-py2.py3-none-any.whl (81kB)
Collecting appdirs
  Downloading https://files.pythonhosted.org/packages/56/eb/810e700ed1349edde4cbdc1b2a21e28cdf115f9faf263f6bbf8447c1abf3/appdirs-1.4.3-py2.py3-none-any.whl
Collecting toml>=0.9.4
  Downloading https://files.pythonhosted.org/packages/a2/12/ced7105d2de62fa7c8fb5fce92cc4ce66b57c95fb875e9318dba7f8c5db0/toml-0.10.0-py2.py3-none-any.whl
Installing collected packages: attrs, click, appdirs, toml, black
Successfully installed appdirs-1.4.3 attrs-19.3.0 black-19.3b0 click-7.0 toml-0.10.0
WARNING: You are using pip version 19.3.1; however, version 20.0.2 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
black \
	--check ./ ../testing \
	--exclude "(testing/scripts/proto|seldon_core/proto/|.eggs|.tox)"
All done! ‚ú® üç∞ ‚ú®
53 files would be left unchanged.
make: Leaving directory '/workspace/source/python'
